auth-web/src/composables/affiliations-factory.ts

```diff
@@ -87,9 +90,11 @@ export const useAffiliations = () => {
       if (!state) return 'Unknown'
       if (state === NrState.INPROGRESS) return NrDisplayStates.DRAFT
       if (state === NrState.APPROVED && (!business.nameRequest.expirationDate)) return NrDisplayStates.PROCESSING
-      else if (business.corpType.code === CorpTypes.INCORPORATION_APPLICATION ||
-              business.corpType.code === CorpTypes.REGISTRATION ||
-              state === NrState.DRAFT) {
+      else if (state === NrState.DRAFT ||
+        [CorpTypes.AMALGAMATION_APPLICATION,
+          CorpTypes.INCORPORATION_APPLICATION,
+          CorpTypes.CONTINUATION_IN,
+          CorpTypes.REGISTRATION].includes(business.corpType.code)) {
```

Comment:
So the old code was missing Amalgamation Application? What was the impact of that? Does this fix a known bug?

Reply:
(This may need a little extra testing from Riyaz. Please call it out in your ticket attention: Riyaz.)